src/index.js

```diff
@@ -1,3 +1,62 @@
-function printToConsole() {
-  console.log("Hello!");
+const heroText = [
+  "measuring traffic density_",
+  "understanding crop health_",
+  "tracking forest loss_",
+  "cell nuclei detection_",
+  "forecasting rip currents_",
+  "prediction of hourly global solar irradiation_",
+  "assessing storm damage_",
+];
+
+function changeHeroTextAndImage(moveForward) {
+  // find visible image id
+  const currentImgElem = document.getElementsByClassName("in-view")[0];
```

Comment:
If I click the backwards / forwards button several times really quickly, I can get the image to disappear without being replaced. I'm not exactly sure why, but I think what is probably happening is that clicking really fast causes some kind of race condition where multiple copies of the fadeIn / fadeOut functions are running and updating the DOM in conflicting ways. The right solution will depend on exactly what's going wrong, but often in these types of cases it's possible to reconfigure things such that the last function to run always "wins", i.e. it overrides any changes that might have been put in place by previous invocations of the same function.

Reply:
I can't seem to reproduce this, but I'll keep an eye on it as I continue to work on the hero component.